Gemfile.lock

```diff
@@ -1,9 +1,9 @@
 GIT
   remote: https://github.com/cloudfoundry/copilot
-  revision: 8d5c57eac99ecb356815dc016392fc342d5d72ef
+  revision: 3cef67a571f546fb36dafcfae7a52e75829721e6
```

Comment:
It is published already (0.0.8) https://rubygems.org/gems/cf-copilot https://github.com/cloudfoundry/copilot/commit/3cef67a571f546fb36dafcfae7a52e75829721e6

Reply:
Can you update our `Gemfile` to point to that release rather than have it pull from Github?
https://github.com/cloudfoundry/cloud_controller_ng/blob/bf0a87532f278caffd966f31b616935049f5dae3/Gemfile#L5

Reply:
Should look something like:
```
gem 'cf-copilot', '~> 0.0.8'
```